app/assets/stylesheets/pageflow/print_view.css.scss

```diff
@@ -112,32 +109,31 @@ body {
     }
 
     h1, h2, h3, p {
-    max-width: initial;
+      max-width: initial;
     }
 
-
     .backgroundArea {
       width: initial;
       height: initial;
       position: absolute;
       background-color: white;
     }
-    .scroller, .content .scroller {
+    .scroller, .content .scroller  {
```

Comment:
Selector should have depth of applicability no greater than 3, but was 5<br>Selector `scroller ` should be written in lowercase with underscores<br>Each selector in a comma sequence should be on its own line